module-2/backend/week_three.md

```diff
@@ -10,14 +10,45 @@ Note: When you're done, submit a PR with a reflection in the comments about how
 ### Questions
 
 1. What is the entry at the command line to create a new rails app?
-2. What do Models generally inherit from in rails?
+rails new  
+
+2. What do Models generally inherit from in rails?  
+ActiveRecord::Base
+
 3. What do Controllers generally inherit from in a rails project?
-4. How would I create a route if I wanted to see a specific horse in my routes fitle assuming I'm sticking to standard conventions and that I didn't want other CRUD functionality?
-5. What rake task is useful when looking at routes, and what information does it give you?
-6. What is an example of a route helper? When would you use them?
-7. What's the difference between what `_url` and `_path` return when combined with a routes prefix?
-8. What are strong params and why are the necessary?
-9. What role does `form_for` play in helping us create our forms?
-10. How does `form_for` know where to submit the user's input?
-11. Create a form using a `form_for` helper to create a new `Horse`. 
-12. Why do we want to validate our models?
+ApplicationController  
+
+4. How would I create a route if I wanted to see a specific horse in my routes fitle assuming I'm sticking to standard conventions and that I didn't want other CRUD functionality?  
+resources :horses, only [:show]  
+
+5. What rake task is useful when looking at routes, and what information does it give you?  
+rake routes  
```

Comment:
Yup! With regard to what does it give you:

* Prefix - building block for your url/path helpers
* Verb - HTTP verb to use when accessing this route
* URI Pattern - Path that a user would enter in their nav bar to access the resource including any wildcards
* Controller#Action - The controller and method on that controller that will handle this request.